chart/spring-eureka-server/templates/configmap.yaml

```diff
@@ -24,9 +25,14 @@ data:
 {{ else }}
           enabled: false
 {{ end }}
+
   application.yaml: |+
     eureka:
+      instance:
+        hostname: localhost
       client:
+        registerWithEureka: false
+        fetchRegistry: false
         serviceUrl:
-          defaultZone: {{ template "eurekaUrl" . }}
+          defaultZone: http://${eureka.instance.hostname}:${server.port}/eureka/
```

Comment:
@jkwong888 I assume this is set in the docker file somewhere?

Reply:
i think spring fills these in based on other properties declared in the yaml